.github/workflows/license_review.yml

```diff
@@ -0,0 +1,24 @@
+name: license_review
+on:
+  pull_request:
+    branches:
+     - main
+    paths:
+      - backend/go.mod
```

Comment:
Should we include `go.sum`?

Reply:
license finder doesn't support it, so it wouldn't have any effect but it couldn't hurt to add it i guess?

Reply:
ah ok, I guess it wouldn't produce anything meaningful even if it did run.

Reply:
yeah it doesn't support it but doesn't hurt to add. added the path